docs/templates/backend.md

```diff
@@ -69,14 +71,14 @@ You can change these settings by editing `~/.keras/keras.json`.
 
 ## Using the abstract Keras backend to write new code
 
-If you want the Keras modules you write to be compatible with both Theano and TensorFlow, you have to write them via the abstract Keras backend API. Here's an intro.
+If you want the Keras modules you write to be compatible with both Theano (`th`) and TensorFlow (`tf`), you have to write them via the abstract Keras backend API. Here's an intro.
```

Comment:
Ok. 
Do I have to use explicit names also for `tensorflow` (i.e. `tensorflow.Variable`rather than `tf.Variable`)?

I basically aligned the two notations consistently (as it was `tf.Variable` and `theano.shared`), and thought that `th` and `tf` are now quite "standard" naming conventions just like `np` is for numpy. :)